src/less/dialog/ds6/dialog-base.less

```diff
@@ -37,29 +37,44 @@
     //-----------------------------
 
     &__window {
-        background-color: @ds6-color-g201-gray;
         left: 0;
         margin: auto;
-        margin-bottom: 15vh;
         min-height: 55px;
+        padding-bottom: 15vh;
         position: absolute;
         right: 0;
         top: 15%;
         will-change: opacity, transform;
         .create-grid(@cols: 8, @gutter: 8px, @size: 7, @min: 6, @max: 8);
 
+        // The background for the dialog window is on this pseudo element.
+        // This was done because `margin-bottom` in safari does not work properly with position absolute elements.
+        &::before {
+            background-color: @ds6-color-g201-gray;
+            bottom: 15vh;
+            content: "";
+            left: 0;
+            position: absolute;
+            right: 0;
```

Comment:
Do we need both `left` and `right`? I also don’t see a width definition, is it inherited from the parent? (Just trying to visualize this from my mobile phone. Ha ha.)

Reply:
Not a huge deal, but I'd go with `left: 0` and `width: 100%` because it's a little clearer to most people.

Reply:
Yup. Readable code always better than smarter code. Reduces cognitive overhead in the future self 😁

Reply:
Yeah, I've never seen that before. 🤷‍♂️

Reply:
I’ve seen both used but if we prefer `width` then I’m fine with switching. Will update the PR in the morning.